roles/openshift_metrics/tasks/cleanup.yaml

```diff
@@ -0,0 +1,19 @@
+---
+- name: remove metrics components
+  command: >
+    {{ openshift.common.client_binary }} -n '{{ project }}'
```

Comment:
Another consideration is we should be relying upon the admin certs and system:admin user since I think this call to the 'oc' binary requires you to be explicitly logged in as system:admin.  Need to discuss the correct path forward with the productization team